localstack/services/events/event_ruler.py

```diff
@@ -43,8 +42,6 @@ def matches_rule(event: str, rule: str) -> bool:
     There is a single static boolean method Ruler.matchesRule(event, rule) -
     both arguments are provided as JSON strings.
     """
-    if config.EVENT_RULE_ENGINE != "java":
```

Comment:
ok to remove this defensive guard to enable using the Java-based event ruler without setting the env variable 👍